src/hammer-vlsi/technology/asap7/defaults.yml

```diff
@@ -34,11 +37,11 @@ vlsi:
       }
     ]
     # This is to get around missing removal_falling in libs for RESET pin ASYNC_DFF.. cells
-    custom_sdc_constraints:
-    - "set_false_path -hold -fall_to [get_db [get_db insts -if {.base_cell.name == ASYNC_DFF*}] .pins -if {.name == *RESET}]"
+    # custom_sdc_constraints:
```

Comment:
Can these comments be deleted as obsolete for now, since there's custom TCL in __init__.py for removing the checks for these cells?

Reply:
Ah, yes they can.